src/css/profile/mobile/common/progress.less

```diff
@@ -14,6 +14,8 @@ tau-progress {
 
 	margin-left: 32 * @unit_base;
```

Comment:
I think that you should change all `unit_base` referring to progress-bar according to this issue or css selector.